hackathon_site/dashboard/frontend/src/slices/order/teamOrderSlice.ts

```diff
@@ -0,0 +1,144 @@
+import {
+    createAsyncThunk,
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+} from "@reduxjs/toolkit";
+import { AppDispatch, RootState } from "slices/store";
+import { get } from "api/api";
+import { APIListResponse, Order, OrderInTable, ReturnOrderInTable } from "api/types";
+import { displaySnackbar } from "slices/ui/uiSlice";
+import { teamOrderListSerialization } from "api/helpers";
+
+interface TeamExtraState {
+    isLoading: boolean;
+    error: null | string;
+    next: string | null;
```

Comment:
Maybe consider changing name to TeamOrderExtraState, just so we don't get confused. Also what is the next field is being used for in this case?

Reply:
originally, i was using "next" in case a team had A LOT of orders (over 100) so it would load the next set from the api, but i doubt a team would have over 100 orders, we should probably block them if that happens.